tests/utils/siteBuilder.js

```diff
@@ -0,0 +1,107 @@
+const { toToml } = require('tomlify-j0.4')
+const fs = require('fs-extra')
+const path = require('path')
+const tempy = require('tempy')
+const os = require('os')
+
+const createSiteBuilder = ({ siteName }) => {
```

Comment:
At some point I would like to add `withUISetting` which will use the API to create the site.